redisearch/client.go

```diff
@@ -68,6 +71,9 @@ func NewClient(addr, name string) *Client {
 // CreateIndex configues the index and creates it on redis
 func (i *Client) CreateIndex(s *Schema) error {
 	args := redis.Args{i.name}
+
+	args = append(args, "MAXTEXTFIELDS", fmt.Sprintf("%d", s.Options.MaxTextFields))
```

Comment:
@Felixls doing this will cause an error. `MAXTEXTFIELDS` should not be proceeded by the number of max text fields. It is only a flag. Quoting docs:

```
MAXTEXTFIELDS: For efficiency, RediSearch encodes indexes differently if they are created with less than 32 text fields. This option forces RediSearch to encode indexes as if there were more than 32 text fields, which allows you to add additional fields (beyond 32) using FT.ALTER.
```
lets remove it :)